tests/tox_env/python/pip/test_pip_install.py

```diff
@@ -35,6 +37,15 @@ def test_pip_install_empty_list(tox_project: ToxProjectCreator) -> None:
     assert execute_calls.call_count == 0
 
 
+def test_pip_install_empty_command_error(tox_project: ToxProjectCreator) -> None:
+    proj = tox_project({"tox.ini": "[testenv]\ninstall_command="})
+    result = proj.run("l")
+    pip = result.state.envs["py"].installer
```

Comment:
You should use `c -k install_command` instead and check that output.

Reply:
Brilliant, that's a very nice feature to have :+1: 

Running that from the CLI currently _succeeds_ (unexpected), and emits:

```
[testenv:py]
install_command = # Exception: ValueError("attempting to parse '' into a command failed")

[testenv:.pkg]
install_command = python -I -m pip install '{packages}'

[tox]
```

(with useful syntax highlighting - the exception-comment message is higlighted in red)

I'm fairly slow, so I'm still determining where the test coverage should be.  It feels to me like perhaps this could be additional (not replacement) coverage, on the `show_config` command.  Guidance appreciated.

Reply:
> Running that from the CLI currently _succeeds_ (unexpected), and emits:

Is expected. Shows the exception message as a side-effect. Also, that output seems outdated. Are you using the latest tox? Can be additional, but note then this test here should not build an entire tox-env, just instantiate tox and work on that.